doc/contents.md

```diff
@@ -0,0 +1,31 @@
+DLite documentation
+===================
+
+This is a temporary table of contents, used during the development phase.
+When completed, the table of contents will be written in pydoc/index.rst
+and built with sphinx.
+
+Table of contents
+-----------------
+
+1. Introduction
+    * What is DLite and what is it used for?
+    * [Features](features.md)
+2. Installation instructions
+3. [Build instructions](build_with_vs.md)
```

Comment:
Here I suggest that we have a main build_instructions.md file with build instructions that we link to here.
The build_with_vs.md file can be referred to from build_instructions.md.

Reply:
Implemented as suggested.